content/tbb/tbb-25/contents.lr

```diff
@@ -5,5 +5,14 @@ title: I'm having a problem with NoScript.
 seo_slug: problem-noscript
 ---
 description: 
-Please see the <mark><a href="https://noscript.net/faq">NoScript FAQ</a></mark>.
-If you believe this is a Tor Browser issue, please report it on our <mark><a href="https://trac.torproject.org/">bug tracker</a></mark>.
+NoScript is a web browser extension.
+It can block or unblock active web content such as JavaScript, Flash and other plugins. 
+By default, several plugins, including Flash, are blocked by NoScript. 
+NoScript provides protection against Cross-site scripting (XSS), CSRF (Cross-site request forgery), clickjacking, man-in-the-middle attacks, and DNS rebinding.
+
+You can disable NoScript by using the "Standard" level in Security Settings (accessible by clicking the small gray shield at the top right of the screen, then selecting "Advanced Security Settings..."). 
+Because JavaScript is an attack vector against the Tor Browser, it is recommended to use the 'Safer' option and most websites will work normally.
+The changes incurred by modifying the Secuirity Level affect all open tabs and are kept even after quitting the browser. 
+
+Please, see the [NoScript FAQ](https://noscript.net/faq) for more problem solutions.
+If you believe this is a Tor Browser issue, please report it on our [bug tracker](https://trac.torproject.org/).
```

Comment:
Maybe better a link to give feedback? https://support.torproject.org/misc/bug-or-feedback/